DolbyIO/Source/Private/DolbyIOConversions.cpp

```diff
@@ -103,6 +108,44 @@ namespace DolbyIO
 		}
 	}
 
+	FString ToString(EDolbyIODeviceDirection Direction)
+	{
+		switch (Direction)
+		{
+			case EDolbyIODeviceDirection::Input:
+				return "Input";
+			case EDolbyIODeviceDirection::Output:
+				return "Output";
+			case EDolbyIODeviceDirection::InputAndOutput:
+				return "Input&Output";
+			case EDolbyIODeviceDirection::None:
+			default:
+				return "None";
+		}
+	}
+
+	FString ToString(enum audio_device::direction Direction)
```

Comment:
You can just define this function as `return ToString(ToEDolbyIODeviceDirection(Direction));`

Reply:
refactored out